pkg/apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -188,6 +188,9 @@ type PipelineSpec struct {
 	// +required
 	InputRefs []string `json:"inputRefs"`
 
+	// Labels lists labels applied to this pipeline
+	// +optional
```

Comment:
nit: add empty line here, same as for other fields

Reply:
Fixed